2021-01-15 10:54:39,581 ERROR: Exception on /explore [GET] [in d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py:1892]
Traceback (most recent call last):
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "D:\marci\DEV\blogpractice\YeetTheMeat\app\routes.py", line 75, in explore
    page, app.config['POSTS_PER_PAGE'], False)
KeyError: 'POSTS_PER_PAGE'
2021-01-15 10:55:33,961 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:45]
2021-01-15 11:27:42,056 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:45]
2021-01-15 11:28:33,180 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:45]
2021-01-15 11:29:06,852 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:45]
2021-01-15 11:35:26,740 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:45]
2021-01-15 12:07:51,433 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 12:25:45,451 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:18:58,197 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:18:58,238 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:18:58,238 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:19:25,378 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:19:25,403 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:19:25,403 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:21:16,211 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:21:16,224 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:21:16,224 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:21:26,500 ERROR: Exception on /reset_password_email_request [GET] [in d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py:1892]
Traceback (most recent call last):
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\marci\DEV\blogpractice\YeetTheMeat\app\routes.py", line 89, in reset_password_request
    title="Reset Password", form=form)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\jinja2\environment.py", line 930, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\jinja2\environment.py", line 883, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\jinja2\environment.py", line 857, in _load_template
    template = self.loader.load(self, name, globals)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\jinja2\loaders.py", line 127, in load
    code = environment.compile(source, name, filename)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\jinja2\environment.py", line 638, in compile
    self.handle_exception(source=source_hint)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\marci\DEV\blogpractice\YeetTheMeat\app\templates\reset_password_request.html", line 10, in template
    {% for error inform.email.errors %}
jinja2.exceptions.TemplateSyntaxError: expected token 'in', got 'inform'
2021-01-15 14:21:53,767 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:21:53,782 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:21:53,782 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:22:00,463 ERROR: Exception on /reset_password_email_request [POST] [in d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py:1892]
Traceback (most recent call last):
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\marci\dev\blogpractice\yeetthemeat\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "D:\marci\DEV\blogpractice\YeetTheMeat\app\routes.py", line 85, in reset_password_request
    send_password_reset_email(user)
  File "D:\marci\DEV\blogpractice\YeetTheMeat\app\email.py", line 18, in send_password_reset_email
    html_body=render_tempalte('email/reset_password.html',
NameError: name 'render_tempalte' is not defined
2021-01-15 14:22:55,969 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:22:55,982 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:22:55,982 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:27:40,811 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:27:41,147 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:27:41,147 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:28:41,054 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:28:41,360 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:28:41,360 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:31:46,058 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:31:46,373 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:31:46,373 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:33:10,933 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:33:11,233 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:33:11,233 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:37:17,015 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:37:17,301 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:37:17,301 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:41:26,732 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
2021-01-15 14:41:27,023 INFO: Microblog startup [in D:\marci\DEV\blogpractice\YeetTheMeat\app\__init__.py:47]
